sass/pages/_scheduler.scss

```diff
@@ -0,0 +1,57 @@
+
+.scheduler-grid {
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+    background: white;
+
+    .scheduler-row {
+        display: table-row;
+
+        .row-info, .row-grid {
+            display: table-cell;
+            border-bottom: 1px solid #f0f0f0;
```

Comment:
Let's add a variables file as well, so we can have these colors all set in a specific location.